### PR TITLE
[ch27871] Expose KR_HOST variable to container

### PR DESCRIPTION
**Pull request by victorpoluceno**
Expose the KR_HOST variable with the host name of the app being deployed.

### CODE REVIEW

File: charts/cf-review-env/values.yaml [hunk 1]
```diff
@@ -32,3 +32,12 @@ customHelmTestURL:
   enabled: false
   args: "/ping/1"
 # Example: https://URL/ping/1
+livenessProbe:
```
Comment (victorpoluceno) [inline charts/cf-review-env/values.yaml:35]: Without these the helm doesn't work. Adding here to make it easier to test.

File: charts/docs/helm-template.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ Ref.: https://helm.sh/docs/chart_template_guide/debugging/
 
 When will be necessary to add new changes on the templates file it's possible to do a debug and then to check if the new change is correct or not, basically you need to run this command:
 
-`helm template test cf-review-env -f values.yaml --debug`
+`helm template test cf-review-env -f cf-review-env/values.yaml --debug \
```
Comment (victorpoluceno) [inline charts/docs/helm-template.md:7]: Adding a complete example that can be tested.

File: charts/cf-review-env/templates/deployment.yaml [hunk 3]
```diff
@@ -34,6 +34,12 @@ spec:
           command: ["{{ .Values.customCommand.command }}"]
           {{- end }}
           {{- end }}
+          env:
+            # Altough the hosts is a list, we always have only one.
+            # We never used this and it will basically be removed in the future.
+            # Sp, it's better to only expose one host.
+            - name: KR_HOST
```
Comment (rodrigo-findhotel) [inline charts/cf-review-env/templates/deployment.yaml:41]: Sorry, but what is KR_HOST?
(it means kubernetes_host 🤔, if yes, you have a good imagination 😄)

**Review by rodrigo-findhotel — APPROVED** [review]

File: charts/cf-review-env/templates/deployment.yaml (hunk 3 above)
Comment (victorpoluceno) [reply charts/cf-review-env/templates/deployment.yaml:41]: ahhaha, nope. it's KR from Kube-Review.

Comment (rodrigo-findhotel) [reply charts/cf-review-env/templates/deployment.yaml:41]: haha, you got me!

**Pull request merged**